scripts/helpers/openai-compatible-agent.py

```
@@ -184,7 +184,7 @@ def main() -> int:
     ap = argparse.ArgumentParser()
     ap.add_argument("--provider", choices=sorted(PROVIDERS), default="generic")
     ap.add_argument("--base-url"); ap.add_argument("--api-key-env"); ap.add_argument("--model")
-    ap.add_argument("--cwd", required=True); ap.add_argument("--max-turns", type=int, default=env_int("OPENAI_COMPAT_MAX_TURNS", 20, 1)); ap.add_argument("--prompt")
+    ap.add_argument("--cwd", required=True); ap.add_argument("--prompt")
```

Ruff (0.16.1)

[error] Multiple statements on one line (semicolon)

(E702)